docker/Dockerfile

```diff
@@ -1,18 +1,15 @@
 # borrowed partly from https://github.com/instantlinux/docker-tools/tree/master/images/proftpd
 
-FROM alpine:3.8
-RUN echo '@edge http://dl-cdn.alpinelinux.org/alpine/edge/main' \
-      >>/etc/apk/repositories && \
-    echo '@testing http://dl-cdn.alpinelinux.org/alpine/edge/testing' \
-      >>/etc/apk/repositories
-RUN apk add --update libcrypto1.1@edge proftpd@testing=1.3.6-r6 proftpd-mod_auth_file@testing=1.3.6-r6 tzdata
-
-RUN apk add --update --no-cache proftpd
+FROM alpine:3.19
+ARG PROFTPD_VERSION=1.3.8b-r0
```

Comment:
is there a use case for having this version as an ARG?  if not, i think we should not expose it, since we do not know if it builds with a different arg.